### ISSUE TITLE
Add the QUERY HTTP method

### ISSUE THREAD

**Issue by 0xflotus** [axios/axios#5465]
### Is your feature request related to a problem? Please describe.

Please consider to support the [HTTP QUERY method](https://www.ietf.org/archive/id/draft-ietf-httpbis-safe-method-w-body-02.html) in the future. 🙏

**Comment by Phenek** [comment]
The community would be very happy to be able to encourage, help, follow this future feature closely.

Let us know @AxiosTeam :)

**Comment by x0rld** [comment]
is there a plan to add it as it assigned ?

**Comment by jasonsaayman** [comment]
yeah sorry been a while i will look at fixing this for any missing methods by the next release
